doc/source/conf.py

```diff
@@ -292,3 +295,15 @@ def download_and_process_files(example_links: List[str]) -> List[str]:
 
     jinja_contexts["examples"] = {"inputs_examples": file_names}
     jinja_contexts["admonitions"] = {"inputs_admonitions": admonitions_links}
+
+
+def revert_gallery_patterns(app, builder):
+    """Revert the gallery excluded patterns."""
+    excluded_gallery_pattern = app.config.exclude_patterns
+    excluded_gallery_pattern.remove("examples/gallery-examples/*.ipynb")
```

Comment:
What if gallery-examples do not exist?
Are we handling the ValueError that will be raised later?